tools/test-npm.sh

```diff
@@ -37,7 +38,7 @@ node cli.js rebuild
 # install npm devDependencies and run npm's tests
 node cli.js install --ignore-scripts
 # run the tests
-node cli.js run-script test-node
+node cli.js run-script test-node -- --reporter=tap | tee ../test-npm.tap
```

Comment:
I'll add a reporter variable for this and the `.bat`.

For the saving, is there a downside to having the `.tap` file left behind? It gets sent to the stdout as well. If there's a reporter option, then I could make it so I only set the `tee` if `--reporter=tap` or `--reporter=junit`.